plugins/CoreHome/javascripts/dataTable.js

```diff
@@ -542,19 +542,21 @@ $.extend(DataTable.prototype, UIControl.prototype, {
             var paddingLeft  = elem.css('paddingLeft');
             paddingLeft      = paddingLeft ? Math.round(parseFloat(paddingLeft)) : 0;
             var paddingRight = elem.css('paddingRight');
-            paddingRight     = paddingRight ? Math.round(parseFloat(paddingLeft)) : 0;
+            paddingRight     = paddingRight ? Math.round(parseFloat(paddingRight)) : 0;
 
-            labelWidth = labelWidth - paddingLeft - paddingRight;
+            if (elem.find('.prefix-numeral').length) {
```

Comment:
When comparing, there is an additional element containing the numeral prefix. Due to the `width: inherit` on the label span, we need to make the element smaller as it should be, as it will get bigger due to that additional element.